lib/internal/Magento/Framework/HTTP/PhpEnvironment/RemoteAddress.php

```diff
@@ -58,6 +58,12 @@ public function getRemoteAddress($ipToLong = false)
             if (!$this->remoteAddress) {
                 $this->remoteAddress = $this->request->getServer('REMOTE_ADDR');
             }
+
+            if ((filter_var($this->remoteAddress, FILTER_VALIDATE_IP) !== true) && (strpos($this->remoteAddress, ',') !== true)) {
```

Comment:
@IcyApril I think it should check 'strpos($this->remoteAddress, ',') == true' to determine if $this->remoteAddress is a comma separated IP list.